codes/typescript/chapter_computational_complexity/leetcode_two_sum.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * @Author: gyt95 (gytkwan@gmail.com) 
```

Comment:
Please set the file header following the format of other JS files.

Reply:
Ok, I made some commits to fix the above mentioned.